README.md

```diff
@@ -1,5 +1,11 @@
 # Fivetran Utilities for dbt
 
+> ## ⚠️ Warning! ⚠️
+>
+> You are viewing a deprecated branch, which is no longer maintained and not recommended for use. This branch remains public to prevent breaking changes to any existing dbt projects that use this package. 
```

Comment:
One small edit suggestion
```suggestion
> You are viewing a deprecated branch, which is no longer maintained and not recommended for use. This branch remains the default branch to prevent breaking changes to any existing dbt projects that use this package. 
```